tests/testGaussianPsfFactory.py

```diff
@@ -50,14 +50,14 @@ def testApply(self):
                             naiveSize += 1
                     else:
                         naiveSize = fixedSize
-                    for minSize in (None, max(1, naiveSize-1), naiveSize, naiveSize + 2):
+                    for minSize in (max(1, naiveSize-1), naiveSize, naiveSize + 2):
```

Comment:
I have traced back the history of this bit of code as far as I could in git, and it lead me to Russell, who indicated that he was actually just editing it from somewhere else when it got put into the oldest file I found it in. I think it was just some poor coding, or at one point it had a use, that was lost though subsequent edits over the years. the None conditions never lead anywhere other than a continue statement, which continues the loop to the next (non None) value, so they are not needed to test this bit of code.